File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocso/OCS1003R02LayR03ListItemInfo.java
package nta.med.data.model.ihis.ocso;

import java.util.Date;

public class OCS1003R02LayR03ListItemInfo {
	private String gwa ;
	private String gwaName ;
	private String bunho ;
	private String suname ;
	private String balheangDate ;
	private String birth ;
	private String naewonDate ;
	private String moveName ;
	private String bunho1 ;
	private Date hopeDate ;
	public OCS1003R02LayR03ListItemInfo(String gwa, String gwaName,
			String bunho, String suname, String balheangDate, String birth,
			String naewonDate, String moveName, String bunho1, Date hopeDate) {
		super();
		this.gwa = gwa;
		this.gwaName = gwaName;
		this.bunho = bunho;
		this.suname = suname;
		this.balheangDate = balheangDate;
		this.birth = birth;
		this.naewonDate = naewonDate;
		this.moveName = moveName;
		this.bunho1 = bunho1;
		this.hopeDate = hopeDate;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getBunho() {
		return bunho;
	}
	public void setBunho(String bunho) {
		this.bunho = bunho;
	}
	public String getSuname() {
		return suname;
	}
	public void setSuname(String suname) {
		this.suname = suname;
	}
	public String getBalheangDate() {
		return balheangDate;
	}
	public void setBalheangDate(String balheangDate) {
		this.balheangDate = balheangDate;
	}
	public String getBirth() {
		return birth;
	}
	public void setBirth(String birth) {
		this.birth = birth;
	}
	public String getNaewonDate() {
		return naewonDate;
	}
	public void setNaewonDate(String naewonDate) {
		this.naewonDate = naewonDate;
	}
	public String getMoveName() {
		return moveName;
	}
	public void setMoveName(String moveName) {
		this.moveName = moveName;
	}
	public String getBunho1() {
		return bunho1;
	}
	public void setBunho1(String bunho1) {
		this.bunho1 = bunho1;
	}
	public Date getHopeDate() {
		return hopeDate;
	}
	public void setHopeDate(Date hopeDate) {
		this.hopeDate = hopeDate;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocso/OCS1003R02LayR03ListItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocso
{
    [Serializable]
    public class OCS1003R02LayR03ListItemInfo
    {
        private String _gwa;
        private String _gwaName;
        private String _bunho;
        private String _suname;
        private String _balheangDate;
        private String _birth;
        private String _naewonDate;
        private String _moveName;
        private String _bunho1;
        private String _hopeDate;

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String GwaName
        {
            get { return this._gwaName; }
            set { this._gwaName = value; }
        }

        public String Bunho
        {
            get { return this._bunho; }
            set { this._bunho = value; }
        }

        public String Suname
        {
            get { return this._suname; }
            set { this._suname = value; }
        }

        public String BalheangDate
        {
            get { return this._balheangDate; }
            set { this._balheangDate = value; }
        }

        public String Birth
        {
            get { return this._birth; }
            set { this._birth = value; }
        }

        public String NaewonDate
        {
            get { return this._naewonDate; }
            set { this._naewonDate = value; }
        }

        public String MoveName
        {
            get { return this._moveName; }
            set { this._moveName = value; }
        }

        public String Bunho1
        {
            get { return this._bunho1; }
            set { this._bunho1 = value; }
        }

        public String HopeDate
        {
            get { return this._hopeDate; }
            set { this._hopeDate = value; }
        }

        public OCS1003R02LayR03ListItemInfo() { }

        public OCS1003R02LayR03ListItemInfo(String gwa, String gwaName, String bunho, String suname, String balheangDate, String birth, String naewonDate, String moveName, String bunho1, String hopeDate)
        {
            this._gwa = gwa;
            this._gwaName = gwaName;
            this._bunho = bunho;
            this._suname = suname;
            this._balheangDate = balheangDate;
            this._birth = birth;
            this._naewonDate = naewonDate;
            this._moveName = moveName;
            this._bunho1 = bunho1;
            this._hopeDate = hopeDate;
        }

    }
}